packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -165,6 +178,20 @@ export type CallCompositeIcons = Partial<
     | 'ParticipantItemScreenShareStart'
     | 'VideoTileMicOff'
     | /* @conditional-compile-remove-from(stable) */ 'LocalCameraSwitch'
+  >
+>;
+
+/* @conditional-compile-remove-from(stable): meeting-composite */
+/**
+ * Icons that can be only overriden in {@link CallWithChatComposite}.
+ *
+ * @beta
+ */
+export type CallWithChatIcons = Partial<
```

Comment:
This will need to list all the icons, but have that covered here: https://github.com/Azure/communication-ui-library/pull/1531/